aqt/updater.py

```diff
@@ -230,18 +229,18 @@ def update(cls, target, base_dir: str, logger):
                 "android_x86",
                 "android_armv7",
             ]:  # desktop version
-                updater.make_qtconf(base_dir, qt_version, arch_dir)
+                updater.make_qtconf(base_dir, target.version, arch_dir)
                 updater.patch_qmake()
                 if target.os_name == "linux":
                     updater.patch_pkgconfig()
                 if versiontuple(target.version) < (5, 14, 0):
                     updater.patch_qtcore(target)
-            elif qt_version.startswith("5."):  # qt5 non-desktop
+            elif Version(target.version) in SimpleSpec(">=5.0,<6.0"):
```

Comment:
We appear to be missing a `from semantic_version import Version` at the top of the file here.